1-engine-intro.ipynb

```diff
@@ -926,14 +937,14 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Howso Prediction Results - Accuracy: 0.82, Precision: 0.79, and Recall: 0.71\n"
+      "Howso Prediction Results - Accuracy: 0.83, Precision: 0.8, and Recall: 0.73\n"
```

Comment:
Do we want to add the "confidence score" somewhere in here?

Reply:
sounds good - what are we calling the categorical_action_probabilities then? (last I checked in the playground, that was our "confidence score")